docs/extensibility/display-section.md

```diff
@@ -725,6 +762,44 @@ ResourceRefs widgets render the lists of links to the associated resources. The
 
 <img src="./assets/display-widgets/ResourceRefs.png" alt="Example of a ResourceRefs widget" style="border: 1px solid #D2D5D9">
 
+### StatisticalCard
+
+StatisticalCard widgets render a card component with a several numerical information elements.
+This widget is primarily designed to be used via [injections](#widget-injections-overview) (**destination: ClusterStats, slot: cards**), allowing the card to be rendered within the dense grid layout of the ClusterOverview's statistical cards section.
+
+#### Example
+
+```yaml
+injections: |-
+  - name: MyTitle
+    widget: StatisticalCard
+    source: $
+    subTitle: MySubtitle
+    value: 112
```

Comment:
Can I use `spec.value` for example as value? It would be more connected to real use cases.

We use `source` as a source for value. Why `StatisticalCard` is different and has `value` field?